src/ClusterBootstrap/deploy.py

```diff
@@ -2830,6 +2830,9 @@ def start_one_kube_service(fname):
         except Exception as e:
             pass
 
+    # recreate the configmap init-user-script each time
+    run_kubectl( ["create configmap init-user-script --from-file=../Jobs_Templete/init_user.sh -o yaml --dry-run | ./deploy/bin/kubectl apply -f -"] )
```

Comment:
how about change it to:
if fname == "userscript":
    run_kubectl( ["create configmap init-user-script --from-file=../Jobs_Templete/init_user.sh -o yaml --dry-run | ./deploy/bin/kubectl apply -f -"] )

Reply:
Done